### PR TITLE
add quicOverride to targetHttpsProxy

### PR DESCRIPTION
**Pull request by danawillow**
I also rearranged the existing attributes there to be alphabetical.

-----------------------------------------------------------------
# [all]
add quicOverride to targetHttpsProxy
## [terraform]
add `quic_override` to `google_compute_target_https_proxy`. Fixes #1365.

### CODE REVIEW

File: products/compute/api.yaml [hunk 1]
```diff
@@ -3084,6 +3084,23 @@ objects:
...
         input: true
         required: true
+      - !ruby/object:Api::Type::Enum
+        name: 'quicOverride'
+        description: |
+          Specifies the QUIC override policy for this resource. This determines
+          whether the load balancer will attempt to negotiate QUIC with clients
+          or not. Can specify one of NONE, ENABLE, or DISABLE. Specify ENABLE to
+          always enable QUIC, Enables QUIC when set to ENABLE, and disables QUIC
+          when set to DISABLE. If NONE is specified, uses the QUIC policy with
```
Comment (ndmckinley) [inline products/compute/api.yaml]: This sentence doesn't quite scan.  I've filed a docs bug, but can you rephrase?

Comment (danawillow) [reply products/compute/api.yaml]: Done

**Comment by ndmckinley** [comment]
I am not a robot, but I also work on MagicModules PRs!

These are some PRs that were opened (and merged) by a previous version of this PR.

depends: terraform-providers/terraform-provider-google#1718
depends: modular-magician/ansible#41
depends: GoogleCloudPlatform/puppet-google-compute#59
depends: GoogleCloudPlatform/chef-google-compute#25
